README.md

```diff
@@ -610,6 +610,148 @@ resources. Once these are created, a client can query for available samples:
 
 #### TODO `GET /Participant/:pid/BiobankSamples
 
+## Organization API
+
+Retrieves metadata about awardees / organizations / sites.
+
+#### `GET /Awardee`
+
+Retrieves a JSON bundle of metadata for all awardees, with nested resources for child 
+organizations and sites within them, representing the full hierarchy of 
+awardees > organizations > sites. No pagination, syncing, or filtering is currently 
+supported on this endpoint.
+
+Example response:
+
+```
+{
+  "entry": [
+    {
+      "fullUrl": "http://localhost:8080/rdr/v1/Awardee/AZ_TUCSON", 
+      "resource": {
+        "displayName": "Arizona", 
+        "id": "AZ_TUCSON", 
+        "organizations": [
+          {
+            "displayName": "Banner Health", 
+            "id": "AZ_TUCSON_BANNER_HEALTH", 
+            "sites": [
+              {
+                "address1": "1234 Main Street", 
+                "address2": "Suite 400", 
+                "adminEmails": [
+                  "bob@example.com", 
+                  "alice@example.com"
+                ], 
+                "city": "Tucson", 
+                "directions": "Turn left on south street.", 
+                "displayName": "Banner University Medical Center - Tucson", 
+                "id": "hpo-site-bannertucson", 
+                "launchDate": "2017-10-02", 
+                "link": "http://www.example.com/", 
+                "mayolinkClientNumber": 7035650, 
```

Comment:
Are these only ever numbers? Do they never start with 0? Just considering whether it should be a string.

Reply:
I believe they are only ever numbers; they are represented as a number in our database at present. I can confirm with Mayo folks if you like.